CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.5.0](https://github.com/jupyter-server/jupyter_server/tree/1.5.0) (2021-02-22)
```

Comment:
The title of the issue says 1.4.1, but here it says 1.5.0 ?

Reply:
Oops, good catch! Fixing now